### DISCUSSION TITLE
Use CDN link on WordPress

### DISCUSSION THREAD

**Discussion by paulmatunog** [master-co/css#68] [Question]
I'd like to build a WordPress website that uses Underscores starter theme and integrate Master CSS to it using the CDN link provided. 

What's your thoughts about this? Best practice?

**Comment by 1aron** [comment]
@paulmatunog There is currently a contributor in Taiwan who have made a [Master CSS installation extension for Wordpress](https://github.com/oberonlai/master-css). Following 2.0's release, related documents and guidelines will be added asap.

**Reply by paulmatunog** [reply]
Thanks @1Aron, but how about just using the CDN?

**Reply by 1aron** [reply]
Of course it's fine to use the CDN directly.
```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <script src="https://cdn.master.co/css"></script>
</head>
<body>
    <h1 class="font:40 font:heavy italic m:50 text:center">Hello World</h1>
</body>
</html>
```
